WinUIGallery/ControlPages/DesignGuidance/IconographyPage.xaml

```diff
@@ -206,6 +208,27 @@
                         <controls:SampleCodePresenter x:Name="XAMLCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
                         <TextBlock Text="C#" />
                         <controls:SampleCodePresenter x:Name="CSharpCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
+                        <TextBlock Text="Tags" />
+                        <ItemsView
+                            x:Name="TagsItemsView"
+                            x:Load="{x:Bind SelectedItem.Tags, Mode=OneWay, Converter={StaticResource emptyCollectionToFalseConverter}}"
+                            ItemsSource="{Binding SelectedItem.Tags, Mode=OneWay}"
+                            Margin="0,4,0,4">
+                            <ItemsView.Layout>
+                                <LinedFlowLayout LineSpacing="4" MinItemSpacing="4" />
+                            </ItemsView.Layout>
+                            <ItemsView.ItemTemplate>
+                                <DataTemplate x:DataType="x:String">
+                                    <ItemContainer>
+                                        <Button Content="{x:Bind Mode=OneWay}" Click="TagButton_Click" />
```

Comment:
I think we should provide a fitting UIA name here to indicate what this button does (imagine what the "..." button is supposed to do).